sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -357,7 +357,9 @@ def get_queue_receiver(
         :keyword Optional[float] max_wait_time: The timeout in seconds between received messages after which the
          receiver will automatically stop receiving. The default value is None, meaning no timeout. If connection
          errors are occurring due to write timing out, the connection timeout value may need to be adjusted. See
-         the `socket_timeout` optional parameter for more details.
+         the `socket_timeout` optional parameter for more details. If there is session enabled queue or topic and 
+         NEXT_AVAILABLE_SESSION is specified, max_wait_time will wait for that time to receive a message from any
+         session before the operations times out.
```

Comment:
nit here + everywhere:
```suggestion
         session before the operation times out.
```